### PR TITLE
Use defaultRangeFilter to decide range_filter exist or not

### PR DESCRIPTION
**Pull request by cydrain**
Signed-off-by: yudong.cai <yudong.cai@zilliz.com>

### CODE REVIEW

File: include/knowhere/config.h [hunk 1]
```diff
@@ -382,13 +382,14 @@ class Config {
     EntryAccess<decltype(PARAM)> PARAM##_access(std::get_if<Entry<decltype(PARAM)>>(&__DICT__[#PARAM])); \
     PARAM##_access
 
+const float default_range_filter = 1.0f / 0.0;
```
Comment (liliu-z) [inline include/knowhere/config.h:385]: nit: Let's follow https://zilliverse.feishu.cn/docs/doccnYCofnA36k6zRy37TX4tMYo use `kCamalCase` for constants.

Comment (cydrain) [reply include/knowhere/config.h:385]: updated

**Review by liliu-z — APPROVED** [review]
/lgtm

**Pull request merged**